src/Files.App/ViewModels/ToolbarViewModel.cs

```diff
@@ -771,7 +771,7 @@ public void SearchRegion_LostFocus(object sender, RoutedEventArgs e)
 			CloseSearchBox();
 		}
 
-		private void SearchRegion_Escaped(object? sender, ISearchBox searchBox) => IsSearchBoxVisible = false;
+		private void SearchRegion_Escaped(object? sender, ISearchBox searchBox) => CloseSearchBox();
```

Comment:
Is this still reachable?

Reply:
Yes, that method is subscribed to an event (Line # 381).
The `Escape` keyboard accelerator was duplicated (`AddressToolbar.xaml` and `SearchBox.xaml`)